packages/l10n/scripts/parse-new-time-zones.js

```diff
@@ -0,0 +1,58 @@
+const readline = require('readline');
```

Comment:
I would suggest to use the [prompts](https://npmjs.com/package/prompts) library, it's easier to handle user prompts.